packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx

```diff
@@ -31,7 +31,10 @@ export function useActiveVisits() {
     '&location=' +
     sessionLocation;
   const url = `/ws/rest/v1/visit?includeInactive=false&v=${customRepresentation}`;
-  const { data, error, isValidating } = useSWR<{ data: { results: Array<Visit> } }, Error>(url, openmrsFetch);
+  const { data, error, isValidating } = useSWR<{ data: { results: Array<Visit> } }, Error>(
+    sessionLocation ? url : '',
```

Comment:
`null` essentially tells SWR "don't run this request". `''` tells SWR "run the fetch with just the configured base url". See [the docs](https://swr.vercel.app/docs/conditional-fetching).

```suggestion
    sessionLocation ? url : null,
```

Reply:
@ibacher thanks so so much for the clear insight.